internal/pkg/keeper/consts.go

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright (C) 2024 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package keeper
+
+const (
+	ApiVersion                      = "v3"
+	ApiBase                         = "/api/v3"
```

Comment:
```suggestion
	ApiBase                         = "/api/" + ApiVersion
```

Reply:
Turned to use the one declared in contract.